src/extensions/AdditiveColormap3DExtensions/index.js

```diff
@@ -0,0 +1,23 @@
+import AdditiveBlendExtension from './AdditiveBlendExtension';
+import MaximumIntensityProjectionExtension from './MaximumIntensityProjectionExtension';
+import MinimumIntensityProjectionExtension from './MinimumIntensityProjectionExtension';
+import BaseExtension from './BaseExtension';
+
+/**
+ * This object contains the BaseExtension, which can be extended for other additive colormap-style (i.e viridis, jet etc.) rendering, as well
+ * implementations of three ray casting algorithms as extensions.
+ * @typedef Extension3D
+ * @type {object}
+ * @property {object} BaseExtension
+ * @property {object} AdditiveBlendExtension
+ * @property {object} MaximumIntensityProjectionExtension
+ * @property {object} MinimumIntensityProjectionExtension
+ */
+const AdditiveColormap3DExtensions = {
+  BaseExtension,
+  AdditiveBlendExtension,
+  MaximumIntensityProjectionExtension,
+  MinimumIntensityProjectionExtension
+};
```

Comment:
Any reason we wouldn't just export these individually?  Creating a single object that is the default export means they cannot be imported individually.

Reply:
I think this was more for name clarity.  The two types of rendering, Additive Colormap and Color Palette, have identical functionalities with Max/Min IP and Additive Blend, so I'm not sure how we would distinguish between them.  We could name the extensions something like `AdditiveColormap3DMaximumIntensityProjectionExtension` but that seemed way too long so I settled on an object export.